### PR TITLE
Closes #82: Fix makemigrations error due to python 2 limitations

### LINKED ISSUE #82: Unable to run initial migrations
I recreated my database VM from scratch. When I attempted to run migrations, there was a warning

```
Your models have changes that are not yet reflected in a migration, and so won't be applied.
Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
```

When I attempted to run `makemigrations` the process returned the following error:
```python
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 377, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/commands/makemigrations.py", line 124, in handle
    self.write_migration_files(changes)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/commands/makemigrations.py", line 152, in write_migration_files
    migration_string = writer.as_string()
  File "/usr/local/lib/python2.7/dist-packages/django/db/migrations/writer.py", line 131, in as_string
    operation_string, operation_imports = OperationWriter(operation).serialize()
  File "/usr/local/lib/python2.7/dist-packages/django/db/migrations/writer.py", line 88, in serialize
    arg_string, arg_imports = MigrationWriter.serialize(arg_value)
  File "/usr/local/lib/python2.7/dist-packages/django/db/migrations/writer.py", line 333, in serialize
    return cls.serialize_deconstructed(path, args, kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/db/migrations/writer.py", line 239, in serialize_deconstructed
    arg_string, arg_imports = cls.serialize(arg)
  File "/usr/local/lib/python2.7/dist-packages/django/db/migrations/writer.py", line 379, in serialize
    % (value.__name__, module_name))
ValueError: Could not find function generate_filename in destinations.models.
```

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

**Comment by CloudNiner** [comment]
@notthatbreezy might want to take a look too

**Comment by notthatbreezy** [comment]
Cool - looks like it fixes the problem

**Comment by CloudNiner** [comment]
Closes #86

File: python/cac_tripplanner/destinations/models.py [hunk 1]
```diff
@@ -3,6 +3,25 @@
...
+    """ Helper for creating unique filenames
+
+    Must be outside the Destination class because makemigrations throws the following error if not:
+        ValueError: Could not find function generate_filename in destinations.models.
+        Please note that due to Python 2 limitations, you cannot serialize unbound method functions
+        (e.g. a method declared and used in the same class body). Please move the function into the
+        main module body to use migrations. For more information, see
+        https://docs.djangoproject.com/en/1.7/topics/migrations/#serializing-values
+
+    Also cannot be a class method of Destination because the function signature must exactly match:
```
Comment (kshepard) [inline python/cac_tripplanner/destinations/models.py]: Interesting. I wonder why it worked when I was testing it.

**Comment by kshepard** [comment]
+1

**Pull request merged**